src/Tests/Entities/ProjectTest.php

```diff
@@ -60,5 +62,38 @@ public function testSuccessToAssociativeArrayFewerProperties()
         $actualOutput = $testProject->toAssociativeArrayFewerProperties();
         $this->assertEquals($expectedOutput, $actualOutput);
     }
+
+    public function testSuccessToAssociativeArrayAllProperties()
+    {
+        $testUserArray = [];
+        $testUserOne = new User(1, 'name', 'avatar', 'role');
+        $testUserTwo = new User(2, 'name', 'avatar', 'role');
+        $testUserThree = new User(3, 'name', 'avatar', 'role');
+        array_push($testUserArray, $testUserOne, $testUserTwo, $testUserThree);
+        $deadline = DateTime::createFromFormat('d/m/Y', '21/12/2012');
+        $testProject = new Project(1, 'name', 1, $deadline, 'client name', 'http://dummyimage.com/200x200.png/ff4444/ffffff', $testUserArray);
+        $expectedOutput = [
+                            'id' => '1', 
+                            'name' => 'name', 
+                            'client_id' => '1', 
+                            'deadline' => '21/12/2012', 
+                            'overdue' => true,
+                            'client_name' => 'client name',
+                            'client_logo' => 'http://dummyimage.com/200x200.png/ff4444/ffffff',
+                            'users' => [['id' => 1, 'name' => 'name', 'avatar' => 'avatar', 'role' => 'role'],
```

Comment:
I think where you have [[, you might want to drop this down onto a new line? Running the vs code formatter will do it right for us!